example/test_problem/Hydro/ParticleTest/Input__Parameter

```diff
@@ -188,6 +188,7 @@ OPT__OUTPUT_TOTAL             1           # output the simulation snapshot: (0=o
 OPT__OUTPUT_PART              0           # output a single line or slice: (0=off, 1=xy, 2=yz, 3=xz, 4=x, 5=y, 6=z, 7=diag) [0]
 OPT__OUTPUT_USER              0           # output the user-specified data -> edit "Output_User.cpp" [0]
 OPT__OUTPUT_PAR_MODE          0           # output the particle data: (0=off, 1=text-file, 2=C-binary) [0] ##PARTICLE ONLY##
+OPT__OUTPUT_PAR_MESH          0           # output the attributes of tracer particles mapped from mesh quantities -> edit "Input__Par_Mesh" [0] ##PARTICLE ONLY##
```

Comment:
I am concerned that users may not realize that `OPT__OUTPUT_PAR_MESH` is enabled by default, and they might forget to include `Input__Par_Mesh`, which could lead to program crashes due to the missing file.

Also, could you remind me if there are any test problems that use tracer particles, so I can update them as well if we decide to enable this feature by default?

Reply:
@hfhsieh 
- We should call `Aux_CheckFileExist()` in `Par_Init_Attribute_Mesh()` to verify the existence of the file `Input__Par_Mesh`. In that case, it should be fine to enable `OPT__OUTPUT_PAR_MESH` by default. Besides, this test problem belongs to a regression test instead of a scientific application, so I don't think it will affect other people's projects. What do you think?
- No other built-in test problem is using tracer particles.

Reply:
@hyschive Agreed. `OPT__OUTPUT_PAR_MESH` is now enabled by default.

Additionally, I have updated `Par_Init_Attribute_Mesh()` to handle corner cases, where `Input__Par_Mesh` is missing or has no fields specified, to avoid segfault.